lldb/source/Plugins/Process/gdb-remote/ProcessGDBRemote.cpp

```diff
@@ -1265,14 +1280,11 @@ Status ProcessGDBRemote::DoResume() {
     } else
       continue_packet_error = true;
 
-    if (continue_packet_error) {
+    if (direction == RunDirection::eRunForward && continue_packet_error) {
       // Either no vCont support, or we tried to use part of the vCont packet
-      // that wasn't supported by the remote GDB server. We need to try and
-      // make a simple packet that can do our continue
-      const size_t num_continue_c_tids = m_continue_c_tids.size();
-      const size_t num_continue_C_tids = m_continue_C_tids.size();
-      const size_t num_continue_s_tids = m_continue_s_tids.size();
-      const size_t num_continue_S_tids = m_continue_S_tids.size();
+      // that wasn't supported by the remote GDB server, or it's the reverse
```

Comment:
It's a bit weird to have this comment here, since this is commenting after an if test that rules out reverse continuing.  If you wanted to describe the general strategy of "first try forward direction: (a)  vCont first, then (b) without vCont then backwards" that should go before you start doing any of this.

Reply:
Removed "or it's the reverse..." from the comment.